dsmr_parser/obis_references.py

```diff
@@ -82,7 +82,7 @@
 BELGIUM_VERSION_INFORMATION = r'\d-\d:96\.1\.4.+?\r\n'
 BELGIUM_EQUIPMENT_IDENTIFIER = r'\d-0:96\.1\.1.+?\r\n'
 BELGIUM_CURRENT_AVERAGE_DEMAND = r'\d-\d:1\.4\.0.+?\r\n'
-BELGIUM_MAXIMUM_DEMAND_MONTH = r'\d-\d:1\.6\.0.+?\r\n'
+BELGIUM_MAXIMUM_DEMAND_MONTH = r'(?<!\()\d-\d:1\.6\.0.+?\r\n' # negative lookbehind on ( to avoid conflict with BELGIUM_MAXIMUM_DEMAND_13_MONTHS
```

Comment:
Would using 
```
^\d-\d:1\.6\.0.+?\r\n
```
not be enough?

Reply:
@koen-dejonghe ? Could you check please?

Reply:
No, `^\d-\d:1\.6\.0.+?\r\n` does not match anything.